dashboard/app/models/pd/workshop.rb

```diff
@@ -2,20 +2,21 @@
 #
 # Table name: pd_workshops
 #
-#  id               :integer          not null, primary key
-#  workshop_type    :string(255)      not null
-#  organizer_id     :integer          not null
-#  location_name    :string(255)
-#  location_address :string(255)
-#  course           :string(255)      not null
-#  subject          :string(255)
-#  capacity         :integer          not null
-#  notes            :text(65535)
-#  section_id       :integer
-#  started_at       :datetime
-#  ended_at         :datetime
-#  created_at       :datetime
-#  updated_at       :datetime
+#  id                 :integer          not null, primary key
+#  workshop_type      :string(255)      not null
+#  organizer_id       :integer          not null
+#  location_name      :string(255)
+#  location_address   :string(255)
+#  course             :string(255)      not null
+#  subject            :string(255)
+#  capacity           :integer          not null
+#  notes              :text(65535)
+#  section_id         :integer
+#  started_at         :datetime
+#  ended_at           :datetime
+#  created_at         :datetime
+#  updated_at         :datetime
+#  processed_location :text(65535)
```

Comment:
Can this be ordered after `location_address` to keep similar fields near each other?